modules/order/order.service.js

```diff
@@ -144,10 +142,15 @@ class OrdersService {
     ).exec();
   }
 
-  async addOrder(data, { user }) {
+  async addOrder(data, user) {
     const { items } = data;
-    const { _id } = user;
-    data.user = { ...data.user, id: _id };
+
+    if (typeof user !== 'undefined') {
```

Comment:
it will not work:
![image](https://user-images.githubusercontent.com/32039216/120774045-ce3eab80-c519-11eb-9125-4b7b4537dc09.png)

Reply:
fixed